sql/2019/12_Mobile_Web/12_14.sql

```diff
@@ -0,0 +1,34 @@
+#standardSQL
+
+# input attributes
+
+# dataset: `httparchive.pages.2019_07_01_mobile`
+# sample: `httparchive.almanac.pages_desktop_1k`
+
+CREATE TEMPORARY FUNCTION parseStructuredData(payload STRING)
+RETURNS ARRAY<STRING> LANGUAGE js AS '''
+  var attrs = [];
+  try {
+    var $ = JSON.parse(payload);
+    var almanac = JSON.parse($._almanac);
+    almanac['input-elements'] && almanac['input-elements'].map(function(node) {
+        Object.keys(node).map(function(attr) {
+            attrs.push(attr);
```

Comment:
Since `tagName` isn't part of the original node's attributes, should we filter that out? It's currently the top attribute in the results.
```suggestion
            if (attr != 'tagName') attrs.push(attr);
```